addons/block_code/code_generation/blocks_catalog.gd

```diff
@@ -302,3 +302,18 @@ static func get_property_setter_block_definition(variable: VariableDefinition) -
 	var block_def := get_variable_setter_block_definition(variable)
 	block_def.description = "Set the %s property" % variable.var_name
 	return block_def
+
+
+static func get_resource_block_definition(file_name: String) -> BlockDefinition:
+	var block_def := BlockDefinition.new()
+
+	# Block Definition's name cannot work with '.'
+	block_def.name = &"get_resource_%s" % file_name.replace(".", "_")
```

Comment:
I feel like this would be better as a fixed block definition that sets the path as an argument since the serialization will handle that just fine. In fact, this is pretty much exactly the same as 4cc4b2845ab95817b0ded577e4107894532887fa with `get_node`. The `BlockExtension` there is neat, but the only really important part is setting the parameter value. I don't think these kinds of dynamically named blocks are generally a good idea since they rely on the serializer to resolve them.

Reply:
Good point!  The block name should be fixed here.  I set it as "get_resource_file_path" in the fixed commit.